tests/unit/test_engine.py

```diff
@@ -635,7 +653,7 @@ def test_bind_non_model(engine):
         engine.bind(object())
 
 
-def test_bind_skip_abstract_models(engine, session, caplog):
+def test_bind_abstract_models(engine, session, caplog):
```

Comment:
why did this test name change if the body didn't?  It looks like abstract models still aren't bound (going by the `caplog.record_tuples` values)